examples/04_scalability.py

```diff
@@ -8,9 +8,12 @@
 
 """
 
+import warnings
```

Comment:
This should be positioned below the relevant comment (ie where it was before :D )